python/foglamp/services/common/utils.py

```diff
@@ -28,7 +29,7 @@
     url_ping = "{}://{}:{}/foglamp/service/ping".format(service._protocol,
                                                         service._address,
                                                         service._management_port)
-    async with aiohttp.ClientSession() as session:
+    async with SESSION or aiohttp.ClientSession() as session:
```

Comment:
its not a correct way!

Reply:
Alternative awaited!

Reply:
My objection is this only! alternative is fake the server, it will make tests more close to real. and no external session injection shall be needed here.